src/server/apsystem.ts

```diff
@@ -522,6 +539,7 @@ export default class ActivityPubSystem {
     const webmention = await this.actorToMention(followerURL, fromActor)
 
     await this.store.forActor(fromActor).followers.add([webmention])
+    await this.store.forActor(fromActor).interacted.add([webmention])
```

Comment:
ah, this means interactions are recorded on Follow, Create and Update, right?

Reply:
that's what I understand, yes

Reply:
This shouldn't be necessary since we track the interacted actor in `approveActivity` and this only gets invoked from there (unless I missed something?)

Reply:
Whoops, yeah, you're right; we were only doing that in "Create" and "Update" before, but we changed it later. Fixed.